### DISCUSSION TITLE
List all segments/sections

### DISCUSSION THREAD

**Discussion by computerality** [Vector35/binaryninja-api#2915] [Q&A] [answered]
Is there some way to get a list of all segments (and sections) in a loaded binary?

**Comment by psifertex** [comment, marked as answer]
~Two~ Three ways: use the "triage view" which is available under the view group drop-down with graph and linear, and by being in linear view and just scrolling to the very top 

Note: neither supports changing permissions via the UI at this time which is a desired feature for 3.2. 

Also, the super boring bv.segments and bv.sections (you can click the addresses in the log to navigate to them still) in the console which I assume doesn't count but I'm just mentioning for completeness sake.

**Reply by computerality** [reply]
Thanks!
Should have scrolled more in triage view :x
Doing `bv.segments`/`bv.sections` in the console is actually a bit nicer since clicking in the triage view seems to always reuse the existing pane
